src/microbe_stage/ChunkConfiguration.cs

```diff
@@ -41,6 +41,11 @@ public struct ChunkConfiguration : IEquatable<ChunkConfiguration>
 
     public Dictionary<Compound, ChunkCompound>? Compounds;
 
+    /// <summary>
+    ///   The type of enzyme needed to break down this chunk.
+    /// </summary>
+    public string DissolverEnzyme;
```

Comment:
Don't you need to modify the Equals and chunk duplication methods as you added a new field?

Reply:
`DamageType` apparently also wasn't included in the Equals, I'll add it too.